internal/executor/options/docker.go

```diff
@@ -0,0 +1,20 @@
+package options
+
+type DockerOptions struct {
+	NoPull       bool
+	NoPullImages []string
```

Comment:
`NoPullImages` sounds a bit weird IMO. What do you think about changing it yo `--docker-skip-pull`?

Reply:
This is similar to the Ansible's flag: https://docs.ansible.com/ansible/latest/dev_guide/testing_integration.html#non-destructive-tests

I think it is a bit easier to type and I've also seen similarly composed clauses in other Docker commands:

```
% docker run --help | grep '\-\-no'
      --no-healthcheck                 Disable any container-specified HEALTHCHECK
```

```
% docker build --help | grep '\-\-no'
      --no-cache                Do not use cache when building the image
```